charts/cryostat/templates/openshiftOauthProxy.tpl

```diff
@@ -16,6 +16,8 @@
     - --tls-cert=/etc/tls/private/tls.crt
     - --tls-key=/etc/tls/private/tls.key
     - --proxy-prefix=/oauth2
+    - --openshift-sar={{ tpl ( .Values.openshiftOauthProxy.accessReview | toJson ) . }}
+    - --openshift-delegate-urls={"/":{{ .Values.openshiftOauthProxy.tokenReview | toJson }}}
```

Comment:
I was thinking about that, but this would mean that it's only authorizing on requests that will be passed to the Cryostat upstream, and not to requests that go to Grafana or S3 or any other future containers that may be configured to be reachable through the proxy. I guess these could all have different token reviews, but it seemed simplest to just keep them everything guarded equally. If there is a case for expanding the configuration options later we can always add that.

Reply:
Or, another option would be to use the same token review for the paths `/api`, `/storage`, and `/grafana`. This way all three current upstream containers are guarded with the same RBAC, but notable `/health` (which also goes to the upstream Cryostat container) would become unguarded, which seems like it makes sense.

Reply:
Sounds good to me! It would be nice to have the key as regular expression but not sure if that's supported in ocp-oauth :D

Reply:
I was looking at the sources earlier and it doesn't look it supports anything other than string prefixes, unfortunately.

Reply:
Ahh opps.. I guess I can have a look and see if I can help them on that haha :D

Reply:
Do you who I should contact? I am unsure who are the maintainers there :D Probably not anytime soon tho until I finish some tasks here :D

Reply:
https://github.com/openshift/oauth-proxy/commits?author=stlaz would be the one to contact, I think.

Reply:
Okay I will try contacting them sometimes soon! Thank youu!!